docker-entrypoint.sh

```diff
@@ -43,9 +43,12 @@ case "$command" in
   ;;
 "test")
   wait-for-it db:5432
+  python manage.py makemigrations --check
```

Comment:
add more resilience: pushing code without migration to production should fail.

If we really need to make a deployment without proper migration (might happen due to kubernetes), the proposal is to manually comment this line for a release as a manual override

Reply:
Good idea!